server/models/procedures.sql

```diff
@@ -1445,4 +1445,113 @@ BEGIN
   WHERE
     sm.document_uuid = document_uuid;
 END $$
+
+-- This processus inserts records relative to the stock movement integration in the posting journal
+CREATE PROCEDURE PostIntegration (
+  IN document_uuid BINARY(16),
+  IN date DATETIME,
+  IN enterprise_id SMALLINT(5) UNSIGNED,
+  IN project_id SMALLINT(5) UNSIGNED,
+  IN currency_id TINYINT(3) UNSIGNED,
+  IN user_id SMALLINT(5) UNSIGNED
+)
+BEGIN
+  DECLARE InvalidInventoryAccounts CONDITION FOR SQLSTATE '45006';
+  DECLARE current_fiscal_year_id MEDIUMINT(8) UNSIGNED;
+  DECLARE current_period_id MEDIUMINT(8) UNSIGNED;
+  DECLARE current_exchange_rate DECIMAL(19, 4) UNSIGNED;
```

Comment:
It doesn't look like you ever use `current_exchange_rate`.  Could you remove it and any other declarations you don't need?